src/defaults.ts

```diff
@@ -31,6 +31,7 @@ export const DEFAULT_THEME: Theme = {
     scrollbarColor: isMacOS ? '' : 'auto',
     selectionColor: 'auto',
     styleSystemControls: true,
+    maxImgPerSec: 30,
```

Comment:
What if some website uses small images, and another has big?

Reply:
They can change it on a per-website-basis that's why I've listed this option under Theme and not under settings.